app/controllers/api/v1/transactions/random_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::V1::Transactions::RandomController < ApplicationController
 
   def show
-    render json: Merchant.random
```

Comment:
oops. this was supposed to have changed a long time ago. Should have designed better tests